cms/templates/settings.html

```diff
@@ -306,6 +306,21 @@ <h2 class="title-2">${_("Requirements")}</h2>
                   <input type="text" class="short time" id="course-effort" placeholder="HH:MM" />
                   <span class="tip tip-inline">${_("Time spent on all course work")}</span>
                 </li>
+                % if settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES') and settings.FEATURES.get('MILESTONES_APP'):
+                <li class="field field-select" id="field-pre-requisite-course">
+                  <form action="#" class="pre-requisite-course-form" method="post">
+                    <label for="pre-requisite-course">${_("Pre-Requisite Course")}</label>
+                    <select class="input" id="pre-requisite-course">
+                        <option value="">None</option>
+                        % for course_info in sorted(pre_requisite_courses, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
+                        <option value="${course_info['course_key']}">${course_info['display_name']}</option>
+                        % endfor
+                    </select>
+                    <span class="tip tip-inline">${_("Minimum level of competency/knowledge prior to taking this course")}</span>
```

Comment:
Suggestion: Courses that you must complete before you begin this course (this text can be used even if there's just one)